### ISSUE TITLE
Problem Changing PDF Viewer

### ISSUE THREAD

**Issue by JasonB579** [lervag/vimtex#1075]
# Problem Changing PDF Viewer

My pdf viewer for vimtex is always Zathura, even though I have it set to Evince in my settings for both Latexmk and Vimtex.

1. Minimal `vimrc` file (actually `init.vim` as I'm using nvim):

    ```vim
    set nocompatible
    filetype off

    let &rtp  = '~/.config/nvim/bundle/Vundle.vim
    call vundle#begin('~/.config/nvim/bundle)

    Plugin 'VundleVim/Vundle.vim'
    Plugin 'lervag/vimtex'

    call vundle#end()
    filetype plugin indent on
    syntax enable

    let g:tex_flavor = "latex"
    let g:vimtex_view_general_viewer = 'evince'
    ```

2. Minimal `.tex` file:

    ```tex
    \documentclass{article}
    \begin{document}

    This is a test

    \end{document}
    ```

3. A concise description of the issue and **the steps to reproduce** it. 

    When I use `\lv` to view the pdf of my `.tex` document I always get Zathura, rather than Evince, which is what I have it set to. This used to work fine but recently (after a reinstall of nvim and all of my plugins) I only get Zathura.

To open the file I use `vim -u init.vim test.tex` in a directory containing only my minimal `init.vim` and the `.tex` related files from `test.tex`
   

4. A `.latexmkrc` file.
    My `~/.latexmkrc` which was modified based on [Issue #175](https://github.com/lervag/vimtex/issues/175) and then, when that didn't work, based on the instructions at the [using latexmk page](http://mg.readthedocs.io/latexmk.html).

    ```
    $pdf_previewer = 'start evince';
    $pdf_update_method = 0;
    $pdflatex = 'pdflatex --shell-escape -synctex=1 %O %S';
    ```
5. Output of `:VimtexInfo`
    ```
    System info
        OS: Linux 4.15.6-1-ARCH
        Vim version: NVIM v0.2.2
        Has clientserver: true
        Servername: /tmp/nvim0ZvS9n/0

    vimtex project: test
        base: test.tex
        root:/home/jason/test
        tex: /home/jason/test/test.tex
        out: /home/jason/test/test.pdf
        log: /home/jason/test/test.log
        aux: /home/jason/test/test.aux
        fls: /home/jason/test/test.fls
        document class: article
        compiler: latexmk
            backend: nvim
            output: /tmp/nvim0ZvS9n/1
            configuration: 
                continuous: 1
                callback: 1
                latexmk options:
                    -verbose
                    -pdf
                    -file-line-error
                    -synctex=1
                    -interaction=nonstopmode
                pid: 5652
                cmd: max_print_line=2000 latexmk -verbose -pdf -file-line-error -synctex=1 -interaction=nonstopmode -pvc -e '$new_viewer_always = "0"' -e '$pdf_previewer = "zathura  -x \"nvr --servername /tmp/nvim98rf23/0 --remote +\%{line} \%{input}\" \%S"' -e '$success_cmd = "'""'nvr'""' --servername '/tmp/nvim98rf23/0' --remote-expr \"vimtex\#compiler\#callback(1)\""' -e '$failure_cmd = "'""'nvr'""' --servername '/tmp/nvim98rf23/0' --remote-expr \"vimtex\#compiler\#callback(0)\""' 'test.tex'
          viewer: Zathura
              xwin id: 0
          qf: LaTeX logfile
              config: 
                  fix_paths: 1
                  packages: 
                      default: 1
                      default: 1
    ```
    I know it says in the `VimtexInfo` that the viewer is Zathura, and that seems to be related to latexmk, but I can't seem to find an answer based in either latexmk or vimtex

**Comment by lervag** [comment]
Are you _really_ doing `nvim -u minivimrc minimal.tex` now? I am suspecting that you are not doing this. If you add `echo "Hello World"` at the end of the minimal vimrc file, do you see the echoed message when starting?

I have a minor suspicion that you have recently switched from Vim to neovim, and that you are confusing configuration files.

But OK, there are a couple of things we can do to find what is happening. After opening a TeX file:

- What is the output of `:verbose let g:vimtex_view_method`?
- What is the output of `:scriptnames`?

I looked at your [dotfiles](https://github.com/JasonB579/dotfiles) repo by cloning it and then doing `rg vimtex`, which shows this:

```
> rg "vimtex"
packages/desktop/latex/latex.sh
18:# Reinstall plugins for vimtex

packages/desktop/latex/config/latex.vim
17:let g:vimtex_fold_enabled = '1'
18:let g:vimtex_fold_manual = '1'
19:let g:vimtex_compiler_progname = 'nvr'
29:let g:vimtex_view_method = 'zathura'

packages/cli/nvim/config/nvim/vundle.vim
71:    Plugin 'lervag/vimtex'
```

This seems to indicate that you are setting your viewer to Zathura...

**Comment by JasonB579** [comment]
I am doing `nvim -u init.vim test.tex` - I added `echo "Hello World"` and it prints before opening the file in vim.

That being said, I found the issue from my dotfiles (I stole them from a friend and missed this change). It turns out that the latex.vim file found in packages/desktop/latex/config/latex.vim is done as a Filetype Plugin and is loaded with any `.tex` file after the init.vim is loaded. I removed that and solved the problem.

Thanks for the help!
